### PR TITLE
PP-4597 output timestamps in miliseconds

### PR DESCRIPTION
**Pull request by georgeracu**
## WHAT YOU DID
Output time in milliseconds
Code cleaup

### CODE REVIEW

File: src/test/java/uk/gov/pay/api/it/GetDirectDebitEventsTest.java [hunk 1]
```diff
@@ -42,19 +42,19 @@
...
-    
+
     @Test
     @PactVerification({"direct-debit-connector", "publicauth"})
     @Pacts(pacts = {"publicapi-direct-debit-connector-get-events"})
     @Pacts(pacts = {"publicapi-publicauth"}, publish = false)
     public void getDirectDebitEvents() {
-        String requestPath = "/v1/events?to_date=2018-03-13T10:00:05Z&from_date=2018-03-13T10:00:03Z&display_size=100&page=1&agreement_id=1";
-        
+        String requestPath = "/v1/events?to_date=2018-03-13T10:00:05.000Z&from_date=2018-03-13T10:00:03.000Z&display_size=100&page=1&agreement_id=1";
```
Comment (alexbishop1) [inline src/test/java/uk/gov/pay/api/it/GetDirectDebitEventsTest.java]: Not sure about this one: this in a input date/time (provided by service users) and for those won’t always have millisecond precision

File: src/test/resources/pacts/publicapi-direct-debit-connector-get-events.json [hunk 2]
```diff
@@ -38,8 +38,8 @@
         "method": "GET",
         "path": "/v1/events",
         "query": {
-          "to_date": ["2018-03-13T10:00:05Z"],
-          "from_date": ["2018-03-13T10:00:03Z"],
+          "to_date": ["2018-03-13T10:00:05.000Z"],
+          "from_date": ["2018-03-13T10:00:03.000Z"],
```
Comment (alexbishop1) [inline src/test/resources/pacts/publicapi-direct-debit-connector-get-events.json]: Again, not sure about changing these

File: src/test/java/uk/gov/pay/api/it/GetDirectDebitEventsTest.java (hunk 1 above)
Comment (georgeracu) [reply src/test/java/uk/gov/pay/api/it/GetDirectDebitEventsTest.java]: Fair point, it slipped through :)

File: src/test/resources/pacts/publicapi-direct-debit-connector-get-events.json (hunk 2 above)
Comment (georgeracu) [reply src/test/resources/pacts/publicapi-direct-debit-connector-get-events.json]: This change is required as we create this request inside `publicapi` and we call `connector` with it. Milliseconds are added in `DateTimeUtils#toUTCDateString` as we use this method everywhere to format ZonedDateTime. We do not receive this parameters from outside world :)

Comment (alexbishop1) [reply src/test/resources/pacts/publicapi-direct-debit-connector-get-events.json]: Ah, OK, I misunderstood where it was coming from

**Pull request merged**